docs/en/SUMMARY.md

```diff
@@ -63,6 +64,7 @@
     * [setProps](api/wrapper-array/setProps.md)
     * [trigger](api/wrapper-array/trigger.md)
     * [update](api/wrapper-array/update.md)
+    * [visible](api/wrapper/visible.md)
```

Comment:
This should be wrapper-array

Reply:
I've made the same as exists() with use the same page.

Reply:
I didn't realize exists uses the same page, this is an error with the exists page.

The wrapper-array pages have different examples that use a wrapper-array

Reply:
Should I fix exists page too ? Or another pull request ?